packages/x-components/src/x-modules/tagging/components/tagging.vue

```diff
@@ -0,0 +1,122 @@
+<script lang="ts">
+import Vue from 'vue';
+import { Component, Inject, Prop, Watch } from 'vue-property-decorator';
+import { xComponentMixin } from '../../../components/x-component.mixin';
+import { SnippetConfig } from '../../../x-installer/api/api.types';
+import { taggingXModule } from '../x-module';
+
+@Component({
+  mixins: [xComponentMixin(taggingXModule)]
+})
+export default class Tagging extends Vue {
+  /**
+   * It injects {@link SnippetConfig} provided by an ancestor as snippetConfig.
+   *
+   * @internal
+   */
+  @Inject('snippetConfig')
+  protected snippetConfig!: SnippetConfig;
```

Comment:
Make this `snippetConfig` optional. We might not always have this snippet config, and the tagging module should still work without it.

```suggestion
  @Inject('snippetConfig')
  protected snippetConfig!: SnippetConfig;
```